### DISCUSSION TITLE
How can I query to determine what version of DuckDB is being created?

### DISCUSSION THREAD

**Discussion by TurboCooler** [Giorgi/DuckDB.NET#99] [Q&A] [answered]
I create a database as follows:

static void create_duckdb_database(string duckdbfile, string test_table_dml_file)
        {
            string table_dml;
            using (var reader = new StreamReader(test_table_dml_file))
            {
                table_dml = reader.ReadToEnd();
            }

            using (var connection = new DuckDBConnection("Data Source=" + duckdbfile))
            {
                connection.Open();

                using (DuckDbCommand command = new DuckDbCommand(table_dml, connection))
                {
                    command.ExecuteNonQuery();
                }
                
            }
        }

When I use QStdudio to try and read the contents of the tables, if fails and says I need a specific version of DuckDB.

How can a query to determine the version of DuckDB file that was created?

**Comment by Giorgi** [comment, marked as answer]
You can use `connection.ServerVersion` to determine DuckDB version.
